lib/core/src/server/build-static.js

```diff
@@ -72,42 +40,105 @@ export async function buildStaticStandalone(options) {
     });
   }
 
-  // compile all resources with webpack and write them to the disk.
-  return new Promise((resolve, reject) => {
-    const previewStartTime = process.hrtime();
-
-    const webpackCb = (err, stats) => {
-      if (err || stats.hasErrors()) {
-        logger.error('Failed to build the storybook');
-        // eslint-disable-next-line no-unused-expressions
-        err && logger.error(err.message);
-        // eslint-disable-next-line no-unused-expressions
-        stats && stats.hasErrors() && stats.toJson().errors.forEach(e => logger.error(e));
-        process.exitCode = 1;
-        return reject(err);
+  logger.info('copy dll..');
+  shelljs.cp(dllPath, path.join(outputDir, 'sb_dll'));
+
+  let startTime;
+
+  return new Promise(res => {
+    logger.info('building manager..');
+    startTime = process.hrtime();
+
+    res();
+  })
+    .then(() =>
+      loadManagerConfig({
+        configType,
+        outputDir,
+        configDir,
+        corePresets: [require.resolve('./manager/manager-preset.js')],
+      })
+    )
+    .then(
+      managerConfig =>
+        new Promise((resolve, reject) => {
+          webpack(managerConfig).run((err, stats) => {
+            if (err || !stats || stats.hasErrors()) {
+              logger.error('Failed to build the manager');
+              // eslint-disable-next-line no-unused-expressions
+              err && logger.error(err.message);
+              // eslint-disable-next-line no-unused-expressions
+              stats && stats.hasErrors() && stats.toJson().errors.forEach(e => logger.error(e));
+              process.exitCode = 1;
+
+              reject(stats);
```

Comment:
if you don't return here it will bother reject and resolve I'm pretty sure

```suggestion
              return reject(stats);
```